### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by ik-web**
- [DEMO LINK](https://ik-web.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ik-web.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,10 +5,50 @@
...
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="logo">
```
Comment (Olexandr6) [inline src/index.html:18]:
```suggestion
        <img src="images/logo.png" alt="Moyo logo">
```

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,50 @@
...
+      </a>
+      <nav class="nav">
```
Comment (Olexandr6) [inline src/index.html:20]:
```suggestion
      </a>
      
      <nav class="nav">
```

File: src/index.html [hunk 3]
```diff
@@ -5,10 +5,50 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны  </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >Ноутбуки и компьютеры</a>
```
Comment (Olexandr6) [inline src/index.html:36]: [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), the 3rd item

File: src/style.css [hunk 4]
```diff
@@ -1 +1,63 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```
Comment (Olexandr6) [inline src/style.css:3]: Always set fonts in the`html {...}` tag

File: src/style.css [hunk 5]
```diff
@@ -1 +1,63 @@
...
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+  list-style: none;
```
Comment (Olexandr6) [inline src/style.css:30]: Move this to `.nav_list`

File: src/style.css [hunk 6]
```diff
@@ -1 +1,63 @@
...
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
```
Comment (Olexandr6) [inline src/style.css:35]: Let's combine them and use only one selector.
Hint: use `:not`

File: src/index.html [hunk 7]
```diff
@@ -5,10 +5,50 @@
...
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">ВИДЕО</a>
```
Comment (Olexandr6) [inline src/index.html:48]: In the css file you use transform: uppercase for this
```suggestion
            <a href="#" class="nav__link">Фото</a>
          </li>
          <li class="nav__item">
            <a href="#" class="nav__link">Видео</a>
```

**Review by Olexandr6 — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (ik-web) [reply src/index.html]: Fixed

File: src/index.html (hunk 2 above)
Comment (ik-web) [reply src/index.html]: Fixed

File: src/index.html (hunk 3 above)
Comment (ik-web) [reply src/index.html]: I fixed it but Im not sure I did it correctly :(

File: src/style.css (hunk 4 above)
Comment (ik-web) [reply src/style.css]: Fixed

File: src/style.css (hunk 5 above)
Comment (ik-web) [reply src/style.css:29]: Fixed

File: src/style.css (hunk 6 above)
Comment (ik-web) [reply src/style.css]: Fixed

File: src/index.html (hunk 7 above)
Comment (ik-web) [reply src/index.html]: Fixed

**Comment by ik-web** [comment]
I've fixed those my mistakes. Could you check it once more please.

**Review by Olexandr6 — APPROVED** [review]
Good job👍

**Pull request closed without merging**